lib/tasks/create_application.rake

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+# NB: there is an issue in Doorkeeper which will cause an "uninitialized constant Doorkeeper::Application" error when
+# running this rake task in production mode. It relates to the lazy-loading of ActiveRecord and the work-around is to call
+# ActiveRecord::Base.logger first. See https://github.com/doorkeeper-gem/doorkeeper/issues/1319
+ActiveRecord::Base.logger
```

Comment:
It seems that this has been fixed in the latest `doorkeeper` gem version, maybe if we bump up our gem to that version it might resolve this automatically? unless they haven't pushed that yet then this might be ok

Reply:
Hi @cwrw, thanks for looking, I'll close this PR and do a separate PR to upgrade the gem: it will need a bit of testing to make sure nothing breaks.